### PR TITLE
Fix gpu tests after pinning bazel version in #1188

### PR DESCRIPTION
**Pull request by ianstenbit**

### CODE REVIEW

File: cloudbuild/README.md [hunk 1]
```diff
@@ -30,14 +30,14 @@ To add a dependency for GPU tests:
 - Have a Keras team member update the Docker image for GPU tests by running the remaining steps
 - Create a `Dockerfile` with the following contents:
 ```
-FROM tensorflow/tensorflow:2.10.0-gpu
+FROM tensorflow/tensorflow:2.11.0-gpu
```
Comment (bhack) [inline cloudbuild/README.md:33]: If you are going to merge this in the Cloud build CI check  https://github.com/keras-team/keras-cv/pull/1170 for alignment.

Comment (ianstenbit) [reply cloudbuild/README.md:33]: Ack -- that LGTM. Merged

Comment (tanzhenyu) [reply cloudbuild/README.md:33]: Do we want 2.11?

Comment (ianstenbit) [reply cloudbuild/README.md:33]: I think yes -- we should be testing against the latest stable release of TF, and offering custom ops compiled against the latest version of TF

**Review by tanzhenyu — APPROVED** [review]

File: cloudbuild/README.md (hunk 1 above)
Comment (bhack) [reply cloudbuild/README.md:33]: See our thread starting at https://github.com/keras-team/keras-cv/pull/1188#issuecomment-1363413573

We are no more a python only library. So we need to start to reason about versioning also at c++ level and probably at runtime that it is still missing but available in TF Addons.

**Pull request merged**